packages/cli-repl/test/e2e-fle.spec.ts

```diff
@@ -44,86 +44,122 @@ describe('FLE tests', () => {
   });
   afterEach(TestShell.cleanup);
 
-  for (const useApiStrict of [ false, true ]) {
-    for (const withSessionToken of [ false, true ]) {
-      // eslint-disable-next-line no-loop-func
-      context(`with AWS KMS (apiStrict=${useApiStrict}, ${withSessionToken ? 'with' : 'without'} sessionToken)`, () => {
-        if (useApiStrict) {
-          skipIfServerVersion(testServer, '< 5.0');
+  function* awsTestCases() {
+    for (const useApiStrict of [ false, true ]) {
+      for (const withSessionToken of [ false, true ]) {
+        for (const withEnvVarCredentials of [ false, true ]) {
+          yield {
+            useApiStrict,
+            withSessionToken,
+            withEnvVarCredentials,
+            testDescription: `with AWS KMS (apiStrict=${useApiStrict}, ` +
+              `${withSessionToken ? 'with' : 'without'} sessionToken, ` +
+              `${withEnvVarCredentials ? 'with' : 'without'} credentials in env vars)`
+          };
         }
+      }
+    }
+  }
```

Comment:
This diff is much easier to view with whitespace – using a generator here to avoid increasing the indentation for the entire test more and more :)